### PR TITLE
Update polkadot js for api-augment

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to update to the latest Polakdot.js

# Review Notes
- Added npm version badges to the readme
- Updated from `optional` to the correct `isOptional` for schema rpcs
- Reorder optional param to be last in order
- Updated polkadotjs api to version 9.2.2

### CODE REVIEW

File: pallets/schemas/src/rpc/src/lib.rs [hunk 1]
```diff
@@ -49,7 +49,7 @@ pub trait SchemasApi<BlockHash> {
 
 	/// validates a schema model and returns `true` if the model is correct.
 	#[method(name = "schemas_checkSchemaValidity")]
-	fn check_schema_validity(&self, at: Option<BlockHash>, model: Vec<u8>) -> RpcResult<bool>;
+	fn check_schema_validity(&self, model: Vec<u8>, at: Option<BlockHash>) -> RpcResult<bool>;
```
Comment (wilwade) [inline pallets/schemas/src/rpc/src/lib.rs:52]: Moving optional to the last param

File: js/api-augment/definitions/schemas.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ export default {
         {
           name: "at",
           type: "BlockHash",
-          optional: true,
+          isOptional: true,
```
Comment (wilwade) [inline js/api-augment/definitions/schemas.ts:10]: Update to isOptional

File: js/api-augment/interfaces/augment-api-consts.ts [hunk 3]
```diff
@@ -143,6 +143,12 @@ declare module '@polkadot/api-base/types/consts' {
        **/
```
Comment (wilwade) [inline js/api-augment/interfaces/augment-api-consts.ts:143]: All of these in the `interfaces` folder are autogenerated

File: js/api-augment/package.json [hunk 4]
```diff
@@ -28,13 +28,13 @@
   "author": "LibertyDSNP",
```
Comment (wilwade) [inline js/api-augment/package.json:28]: Version updates

**Review by saraswatpuneet — APPROVED** [review]

**Pull request merged**